celery_serverless/worker_management.py

```diff
@@ -6,11 +6,16 @@
 
 import celery.bin.celery
 import celery.worker.state
-from celery.signals import celeryd_init, worker_ready
+from celery.signals import celeryd_init, worker_ready, task_prerun, task_postrun
+from celery.exceptions import WorkerShutdown
 
 logger = logging.getLogger(__name__)
 logger.setLevel('DEBUG')
 
+# To store the Worker instance.
+# Sometime it will change to a Thread or Async aware thing
+context = {}
```

Comment:
This makes me think of a class instance that keeps its own context.

Reply:
Estive pensando. Não funciona, pq os sinais já são compartilhados entre todos os workers do processo.

Represar esse `context` oferece a falsa impressão de que seria possível ter o "maquinário" completo 2+ vezes simultaneamente no mesmo processo, o que não é verdade.